lib/cuca/app.rb

```diff
@@ -108,12 +108,10 @@ def initialize(app_path: nil, log_path: nil, public_path: nil, additional_suppor
 
 
             #Custom exceptions logger
-            if App.config['exceptions_log'] then
-                @exception_logger = Logger.new("#{@log_path}/exceptions") 
-                @exception_logger.level = Logger::ERROR
-                @exception_logger.formatter = proc do |severity, datetime, progname, msg|
-                    "#{msg}"
-                end
+            @exception_logger = Logger.new((App.config['exceptions_log'] ? "#{@log_path}/exceptions" : "/dev/null")) 
```

Comment:
I'd prefer to pass the logger object directly with App.config['excepsion_log'], then it can also be configured as a user likes.
Example:
@exception_logger = Logger.new('/dev/null')

if App.config['exception_log'] then 
   raise  CucaException, "exception_log needs to be kind of Logger" unless App.config['excepion_log'].kind_of?(Logger)
   @exception_logger = App.config['exception_log']
end

Reply:
Very good suggestion. Implemented with commit 61d9c10b21058a10a32abce9ab8f666991136184